src/components/projects/Projects.jsx

```diff
@@ -1,65 +1,86 @@
-import Paper from "@material-ui/core/Paper";
-import Table from "@material-ui/core/Table";
-import TableBody from "@material-ui/core/TableBody";
-import TableCell from "@material-ui/core/TableCell";
-import TableContainer from "@material-ui/core/TableContainer";
-import TableRow from "@material-ui/core/TableRow";
+import DescriptionRoundedIcon from "@material-ui/icons/DescriptionRounded";
 import getProjects from "./ProjectsAPI.js";
-import { makeStyles } from "@material-ui/core/styles";
+import { useHistory } from "react-router-dom";
+import {
+  Button,
+  Paper,
+  TableCell,
+  TableContainer,
+  TableRow,
+  Typography,
+} from "@material-ui/core";
 import React, { useEffect, useState } from "react";
 import "./Projects.scss";
 
-const useStyles = makeStyles({
-  table: {
-    maxWidth: "100%",
-  },
-});
-
 function ProjectList() {
-  const classes = useStyles();
   const [projects, setProjects] = useState([]);
 
   useEffect(() => {
     async function data() {
       const response = await getProjects();
-      setProjects(response);
+      if (response === true) {
+        return;
+      } else {
+        setProjects(response);
+      }
     }
     data();
   }, []);
 
+  const history = useHistory();
+
+  function handleEntryClick() {
+    history.push("/projects/entry");
+  }
+  function handleViewClick(id) {
+    history.push(`/projects/${id}`);
+  }
   return (
     <div className="Projects-Container">
-      <Button color="primary" variant="outlined">
+      <Button color="primary" onClick={handleEntryClick} variant="contained">
         New Project
       </Button>
 
       <div className="Projects-List">
         <TableContainer component={Paper}>
-          <Table aria-label="simple table" className={classes.table}>
-            <TableBody>
-              {projects.length === 0 ? (
-                <TableRow key="1">
-                  <TableCell component="th" scope="row">
-                    <img
-                      alt=""
-                      className="empty-file"
-                      src="https://static.thenounproject.com/png/469473-200.png"
-                    />
-                  </TableCell>
-                </TableRow>
-              ) : (
-                projects.map((project) => (
-                  <TableRow key={project.id}>
-                    <TableCell component="th" scope="row">
+          {projects.length === 0 ? (
+            <div className="empty-file">
+              <DescriptionRoundedIcon
+                style={{ color: "#e2e2e2", fontSize: "10rem" }}
+              />
+              <Typography
+                style={{
+                  color: "#d2d2d2",
+                  fontSize: "1.2rem",
+                  fontWeight: "bold",
+                }}
+              >
+                Looks like, there are no projects
+              </Typography>
+            </div>
+          ) : (
+            projects.map((project) => (
+              <TableRow key={project.id}>
```

Comment:
In this case, I think is not necessary to use key

Reply:
Ok, will change it right away